### PR TITLE
149 pylib docs and restructure 

### PR DESCRIPTION
**Pull request by GwennanS**
Closes GH-149
Closes GH-173
did TODOs of py_scc
docs of py_scc
restructure to 
cc_library/
├── example_scripts_js_scc/
│/////├── display_angular
│/////├── display_node
├── example_scripts_py_scc/
│/////├── controlboard.py
│/////├── ...
├── js_scc/
└── py_scc/
////// └── sciler/
////// ////// ├── app.py
////// ////// ├── ...
////// ////// └── device.py

also #194

### CODE REVIEW

**Comment by mrlomar** [comment]
Could you rename js-scc to js_scc for consistency with py_scc?
Could you move and rename /py_scc/sciler/scripts to /example_scripts_py_scc?
Could you rename /example-scripts-js-scc to js_scc for consistency with /example_scripts_py_scc?
Is it possible then to merge the contents of /py_scc/sciler and /py_scc/sciler/scclib into /py_scc?

File: cc_library/py_scc/sciler/scclib/app.py [hunk 1]
```diff
@@ -193,6 +211,9 @@ def __on_message(self, client, userdata, message):
         This method is called when the client receives
          a message from the broken for a subscribed topic.
         The message is printed and send through to the handler.
+        :param client: mqtt client
+        :param userdata: the private user data as set in Client() or userdata_set()
+        :param message: message from the back-end
```
Comment (mrlomar) [inline cc_library/py_scc/sciler/scclib/app.py]: Why use sometimes @:param en somethings just  :param?

File: cc_library/py_scc/README.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+This is the library to create devices to work together with the SCILER system 
+
+### Installation
+- pip install with ```pip install scclib```
+
+### Using this library
+- import lib with `from scclib.device import Device`
+##### create a class that extends `Device`
+- in order to do this:
+    - implement getStatus() which should return a dictionary of the current status
```
Comment (IssaHanou) [inline cc_library/py_scc/README.md]: maybe method names in `..`

File: cc_library/example_scripts_py_scc/controlboard_config.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "id": "controlBoard",
+  "description": "Control board with three switches, three slides with lights and one main switch.",
```
Comment (mrlomar) [inline cc_library/example_scripts_py_scc/controlboard_config.json]: the description is unnecessary

File: cc_library/example_scripts_py_scc/display_config.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+  "id": "display",
+  "description": "Display can print hints",
```
Comment (mrlomar) [inline cc_library/example_scripts_py_scc/display_config.json]: remove description

File: cc_library/example_scripts_py_scc/door_config.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{
+  "id": "door",
+  "description": "Door can open and close",
```
Comment (mrlomar) [inline cc_library/example_scripts_py_scc/door_config.json]: remove description

File: cc_library/example_scripts_py_scc/hue_lights_config.json [hunk 6]
```diff
@@ -0,0 +1,6 @@
+{
+  "id": "hue_lights",
+  "description": "4 spotlights",
```
Comment (mrlomar) [inline cc_library/example_scripts_py_scc/hue_lights_config.json]: remove description

File: cc_library/example_scripts_py_scc/instruction_soundboarrd.md [hunk 7]
```diff
@@ -0,0 +1,114 @@
+# Soundboard specifics
```
Comment (mrlomar) [inline cc_library/example_scripts_py_scc/instruction_soundboarrd.md]: filename contains one `r` to much

File: cc_library/example_scripts_py_scc/keypad.json [hunk 8]
```diff
@@ -0,0 +1,7 @@
+{
+  "id": "keypad",
+  "host": "192.168.178.82",
+  "port": 1883,
+  "labels": []
+
```
Comment (mrlomar) [inline cc_library/example_scripts_py_scc/keypad.json]: remove enter

File: cc_library/py_scc/sciler/app.py [hunk 9]
```diff
@@ -49,19 +56,27 @@ def __init__(self, config, device):
 
     def __on_log(self, level, buf):
         """
-        MQTT Client method.
+         MQTT Client method.
```
Comment (mrlomar) [inline cc_library/py_scc/sciler/app.py]: Why this space?

File: cc_library/py_scc/sciler/app.py [hunk 10]
```diff
@@ -207,8 +233,9 @@ def __on_message(self, client, userdata, message):
     def __handle(self, message):
         """
         Interpreter of incoming messages.
-        Correct sciler mapper is called with the content of the message.
+        Correct  mapper is called with the content of the message.
```
Comment (mrlomar) [inline cc_library/py_scc/sciler/app.py]: one space too much between correct and mapper

**Pull request merged**